lobster/tools/json/json.py

```diff
@@ -107,15 +104,15 @@ def process_tool_options(self, options, work_list):
 
     @classmethod
     def process(cls, options, file_name):
-        with open(file_name, "r", encoding="UTF-8") as fd:
-            data = json.load(fd)
-
-        # First we follow the test-list items to get the actual data
-        # we're interested in.
         try:
+            with open(file_name, "r", encoding="UTF-8") as fd:
+                data = json.load(fd)
             data = get_item(root     = data,
                             path     = options.test_list,
                             required = True)
+        except UnicodeDecodeError as decode_error:
+            print("%s: File is not encoded in utf-8: %s" % (file_name, decode_error))
```

Comment:
I observed that the old string formatting method  is used rather than f-strings, is there any specific reason due to which it is preferred?

Reply:
No, there is no particular reason. I just used it because it is used everywhere else, to stay consistent.

Reply:
This is something we should address in the coding guideline https://github.com/bmw-software-engineering/lobster/pull/88